test/Common.FunctionalTests/MultiApplicationTests.cs

```diff
@@ -43,6 +43,8 @@ public async Task FailsAndLogsWhenRunningTwoInProcessApps()
         {
             var parameters = _fixture.GetBaseDeploymentParameters(HostingModel.InProcess, publish: true);
             parameters.ServerConfigActionList.Add(DuplicateApplication);
+            parameters.GracefulShutdown = true;
```

Comment:
Why is this required?

Reply:
Treat StopServer as a request for GracefulShutdown

Reply:
> Treat StopServer as a request for GracefulShutdown

Another PR IMO. Looking into why graceful shutdown is needed.